integration-tests/ledger-signer.spec.ts

```diff
@@ -4,6 +4,8 @@ import TransportNodeHid from "@ledgerhq/hw-transport-node-hid";
 import { ligoSample } from "./data/ligo-simple-contract";
 import { TezosToolkit } from '@taquito/taquito';
 
+// PLEASE NOTE MAY NEED TO TEST ONE TEST AT A TIME
```

Comment:
could you elaborate more on this comment?

Reply:
mainly for the error i would get for 
```
[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "PublicKeyRetrievalError: Unable to retrieve Public Key from Ledger".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```

Reply:
have you tried wrapping the asynchronous parts of your test with a try-catch block? I think that might help with those errors